### PR TITLE
Fix "Withdraw from PrivateKey" layout

### LINKED ISSUE #193: Withdraw from private key dialogue is not formatted correctly
## Scope

- The component WithdrawFromPrivate.js is since recently oddly formatted
Screen Shot 2019-07-08 at 16 16 27

## Deliverables

- Format it such that it looks good again
- Use rimble-ui components everywhere and replace all bootstrap components

## Roles
bounty gardener: @TimDaub / 10%
bounty worker: name / 75%
bounty reviewer: name / 15%

### PR DESCRIPTION
**Pull request by MaxStalker**
Fixes #193 
Ping me if you will have troubles testing this. I know how to trick App to render this specific view :)

Visual confirmation that solution works:

**Desktop**:
image

**Mobile:**
image

### CODE REVIEW

**Comment by vrde** [comment]
I'm reviewing it! (Cannot add myself to the reviewers)

File: src/components/WithdrawFromPrivate.js [hunk 1]
```diff
@@ -164,49 +169,40 @@ export default class SendToAddress extends React.Component {
...
-                  placeholder={currencyDisplay(0)}
-                  value={this.state.amount}
-                  onChange={event => this.updateState('amount', event.target.value)} />
-              </div>
-              {products}
-            </div>
-            <Button
-              size={'large'}
+          <Field mb={3} label={i18n.t('withdraw_from_private.from_address')}>
+            <Input
```
Comment (vrde) [inline src/components/WithdrawFromPrivate.js]: A really minor thing, the address is spell checked.

image

Consider adding `spellcheck="false"`.

Comment (TimDaub) [reply src/components/WithdrawFromPrivate.js]: @vrde is this a problem with all `<Input />`s? If so, we should consider filing an issue [here](https://github.com/ConsenSys/rimble-ui/issues).

Comment (MaxStalker) [reply src/components/WithdrawFromPrivate.js]: @TimDaub that's not a bug, per se, just a specific use case, where we don't need spellcheck enabled :)

**Comment by MaxStalker** [comment]
Thank you, @TimDaub !

**Pull request merged**